integration/common.sh

```diff
@@ -23,23 +23,46 @@ function runTest() {
 function waitForServerStartup() {
   # The server will 404 the request as there's no handler for it. This error doesn't matter
   # as the test will fail if the server is really not up.
-  PORT=$1
+  local PORT=$1
   wget -q --spider --retry-connrefused --waitretry=1 -t ${STARTUP_WAIT_SECONDS} localhost:${PORT}
   # Wait a bit more to give it a chance to become actually available e.g. if Travis is slow
   sleep 2
 }
 
+function killPid() {
+  local pid=$1
+  set +e
+  while kill -INT ${pid} > /dev/null
+  do
+    sleep 1
+    ((count++))
```

Comment:
nit: Define count outside the loop, so it gets properly scoped?

local count=0
while ...; do
  # ...
done